### PR TITLE
MAJOR PATCH ADDED MINECRAFT DIAMONDS

### PR DESCRIPTION
**Pull request by mudkipslaps**
Patch Notes: Trophy Weapons
-ADDED MINECRAFT DIAMONDS
-Shotgun Changed to a pump action, increased the mag size by 6 (Dont worry, a proper double barrel is coming soon)
-added the diamond pickaxe, a glittery alternative to the pickaxe
-added the monowhip, a high tech small game weapon obtainable via crafting
-added the sniper rifle obtainable from killing the desert raiders
-buffed the minigun (cost reduced by .5, damage buffed by .05)
https://www.youtube.com/watch?v=DeO_vuhMM9o

### CODE REVIEW

File: ew/static/smelting.py [hunk 1]
```diff
@@ -336,7 +365,8 @@
         ],
         ingredients={
             ewcfg.item_id_monsterbones: 100,
-            ewcfg.item_id_faggot: 1
+            'scalp':100,
```
Comment (aristotlepenguin) [inline ew/static/smelting.py:368]: as a rule, we wouldn't use souls as crafting materials with the possible exception of some in-game events. please change this back

File: ew/static/items.py [hunk 2]
```diff
@@ -485,10 +485,23 @@
     EwGeneralItem(
         id_item=ewcfg.item_id_bloodstone,
         str_name="blood stone",
-        str_desc="Formed from the cracking of monster bones, it glistens in your palm with the screams of those whos bones comprise it. Perhaps it will be of use one day.",
+        str_desc="Formed from the cracking of monster bones, the flesh of your fellow gangster, and a soul, it glistens in your palm with the screams of those whos bones comprise it.",
```
Comment (aristotlepenguin) [inline ew/static/items.py:488]: see below, we're not doing this

File: ew/static/smelting.py [hunk 3]
```diff
@@ -211,6 +211,35 @@
         },
         products=[ewcfg.weapon_id_pickaxe]
     ),
+    EwSmeltingRecipe(
```
Comment (aristotlepenguin) [inline ew/static/smelting.py:214]: i don't see any code showing where this modifies mining yields, pretty funny if intentional but do you want to make the diamond pick useless?

File: ew/static/weapons.py [hunk 4]
```diff
@@ -1506,8 +1505,8 @@ def wef_harpoon(ctn = None):
         str_equip="You equip the model 397 hunting rifle.",
         str_name="model 397 hunting rifle",
         str_weapon="a model 397 hunting rifle",
-        str_weaponmaster_self="You are a rank {rank} gentleman of the model 397 hunting rifle.",
```
Comment (aristotlepenguin) [inline ew/static/weapons.py:1509]: lol, did you just find the whole 'gentleman' thing cringe? there are other weapons that use unique titles. i could take it or leave it, just curious on your reasoning

Comment (mudkipslaps) [reply ew/static/weapons.py:1509]: No, it just doesnt make sense given that it isnt a hunting rifle anymore in terms of stats

File: ew/static/smelting.py (hunk 3 above)
Comment (mudkipslaps) [reply ew/static/smelting.py:214]: ew/cmd/juviecmd/juviecmds.py line 419 does it afaik

File: ew/static/smelting.py (hunk 1 above)
Comment (mudkipslaps) [reply ew/static/smelting.py:368]: Its allready a crafting material on reanimated corpses, will change for now

Comment (aristotlepenguin) [reply ew/static/smelting.py:368]: true, but you can pull souls back out of those. the idea is that there shouldn't be an incentive to permanently destroy them

File: ew/static/smelting.py (hunk 3 above)
Comment (aristotlepenguin) [reply ew/static/smelting.py:214]: pffff didn't see it my bad

**Pull request merged**